tutorials-and-examples/inference-servers/checkpoints/checkpoint_converter.sh

```diff
@@ -64,37 +72,146 @@ download_huggingface_checkpoint() {
         fi
     elif [[ $MODEL_NAME == *"gemma"* ]]; then
         TOKENIZER_PATH=/base/tokenizer.model
+        if [[ $MODEL_PATH == *"gemma-2b-it-pytorch"* ]]; then
+            huggingface-cli download google/gemma-2b-pytorch config.json --local-dir ${INPUT_CKPT_DIR_LOCAL}
+        fi
     else
         echo -e "Unclear of tokenizer.model for ${MODEL_NAME}. May have to manually upload."
     fi
 }
 
+download_meta_checkpoint() {
+    META_URL=$1
+    MODEL_PATH=$2
+    echo -e "$META_URL" | llama download --source meta --model-id $MODEL_PATH
+}
+
 convert_maxtext_checkpoint() {
     BUCKET_NAME=$1
-    MODEL_NAME=$2
-    VARIATION_NAME=$3
-    MODEL_SIZE=$4
-    MAXTEXT_VERSION=$5
-
-    if [ -z $MAXTEXT_VERSION ]; then
-        MAXTEXT_VERSION=jetstream-v0.2.2
+    MODEL_PATH=$2
+    MODEL_NAME=$3
+    OUTPUT_CKPT_DIR=$4
+    VERSION=$5
+    HUGGINGFACE=$6
+    META_URL=$7
+
+    echo -e "\nbucket name=${BUCKET_NAME}"
+    echo -e "\nmodel path=${MODEL_PATH}"
+    echo -e "\nmodel name=${MODEL_NAME}"
+    echo -e "\nversion=${VERSION}"
+    echo -e "\noutput ckpt dir=${OUTPUT_CKPT_DIR}"
+    echo -e "\nhuggingface=${HUGGINGFACE}"
+    echo -e "\nurl=${META_URL}"
+
+    if [ -z $VERSION ]; then
+        VERSION=jetstream-v0.2.2
     fi
 
     git clone https://github.com/google/maxtext.git
 
     # checkout stable MaxText commit
     cd maxtext
-    git checkout ${MAXTEXT_VERSION}
+    git checkout ${VERSION}
     python3 -m pip install -r requirements.txt
+
+    if [ $VERSION == "jetstream-v0.2.2" || $VERSION == "jetstream-v0.2.1" || $VERSION == "jetstream-v0.2.0" ]; then
```

Comment:
$VERSION needs to be quoted, bash is trying to execute it as a command:

line 120: [: missing `]'
line 120: experimental_proxy: command not found

Reply:
and also need to use [[ ]]